src/kvirc/kernel/KviIrcConnectionServerInfo.cpp

```diff
@@ -275,6 +273,566 @@ KviBasicIrcServerInfo::~KviBasicIrcServerInfo()
+		case 'z': return __tr2qs("z: Recipient for OPERWALL messages"); break;
+	}
+	return KviBasicIrcServerInfo::getUserModeDescription(mode);
+}
+
+const QString & KviHyperionIrcServerInfo::getUserModeDescription(QChar mode)
+{
+	switch(mode.unicode())
+	{
+		case 'A': return __tr2qs("A: Can see all servers and IP addresses"); break;
+		case 'B': return __tr2qs("B: Allows the use of SETHOST, SETNAME, and SETIDENT commands on other people"); break;
+		case 'C': return __tr2qs("C: Prevent CTCPs"); break;
+		case 'D': return __tr2qs("D: Allows the use of DIE and RETART"); break;
+		case 'E': return __tr2qs("E: Receive private messages from registered nicks only"); break;
+		case 'F': return __tr2qs("F: Immune to flood protection"); break;
+		case 'G': return __tr2qs("G: Allows the use of KILL for users on remote servers"); break;
+		case 'H': return __tr2qs("H: Allows the use of REHASH"); break;
+		case 'I': return __tr2qs("I: Prevents you from receiving INVITEs"); break;
+		case 'K': return __tr2qs("K: Allows the use of KLINE"); break;
+		case 'L': return __tr2qs("L: Allows the use of 4-argument LUSERS to force a recount"); break;
+		case 'M': return __tr2qs("M: Allows the use of NOTICE based on mask"); break;
+		case 'N': return __tr2qs("N: Can override Q-Lines"); break;
+		case 'P': return __tr2qs("P: Allows the use of SETHOST, SETIDENT and SETNAME"); break;
+		case 'Q': return __tr2qs("Q: Prevents you from being affected by channel forwarding"); break;
+		case 'R': return __tr2qs("R: Allows the use of CONNECT, SQUIT, and HTM"); break;
+		case 'S': return __tr2qs("S: Allows the use of remotely running server commands"); break;
+		case 'T': return __tr2qs("T: Allows you to appear in /stats p"); break;
+		case 'U': return __tr2qs("U: Allows the use of UNKLINE"); break;
+		case 'V': return __tr2qs("V: Allows the use of MAP, LINKS, as well as receiving connection information");
+		case 'W': return __tr2qs("W: Allows the ability to send WALLOPS"); break;
+		case 'X': return __tr2qs("X: Grants access to experimental features"); break;
+		case 'Y': return __tr2qs("Y: Displays extra information during netjoins"); break;
+		case 'Z': return __tr2qs("Z: Allows the ability to send OPERWALL messages"); break;
+		case 'a': return __tr2qs("a: Can see all users and channels"); break;
+		case 'b': return __tr2qs("b: Recipient for spam warning notices"); break;
+		case 'c': return __tr2qs("c: Recipient for connect and netjoin notices"); break;
+		case 'e': return __tr2qs("e: Registered"); break;
+		case 'f': return __tr2qs("f: Recipient for full I-Line notices"); break;
+		case 'h': return __tr2qs("h: Sets user in high priority mode"); break;
+		case 'j': return __tr2qs("j: Allows the use of auto D-Lines"); break;
+		case 'l': return __tr2qs("l: Recipient for new channel creation notices"); break;
+		case 'm': return __tr2qs("m: Cannot be KICKed, DEOPed, or D-Lined"); break;
```

Comment:
```Cannot be KICKed, DEOPed, or D-Lined```

This is not technically wrong but... I would use Can't instead of Cannot, again its a consistency thing with other areas or program not here specific.